### PR TITLE
Socks

### PR DESCRIPTION
**Pull request by orta**
#250

### CODE REVIEW

File: views/socks.slim [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+          markdown:
+            CocoaPods was started in 2011. We are a community run project, with no large-scale corporate backing. We're often asked if we'd do something like t-shirts or hoodies. 
+            
+            We don't want to use CocoaPods' popularity to make money. So, we are putting all profits into a charity.
+        .col-sm-2
+        .col-sm-4
+          p
+            pre
+              <code class="highlight ruby"><span class="n">project</span> <span class="s1">'the world'</span></br>
+                <span class="n">target</span> <span class="s1">'My Feed’ </span><span class="k">do</span><br/>
```
Comment (dbgrandi) [inline views/socks.slim]: Should this be `My Feet`?

File: views/socks.slim [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+
+        .col-md-5.col-md-pull-5
+          markdown:
+            <p>We know a <a src="http://www.buzzfeed.com/jessicamisener/buzzfeet#.xu2baWQ4X" target="_blank">bubble</a> when we see one. We see a great opportunity to get in early on the Cocoa Socks business before Apple comes in and deprecates us with the Socks Packaging Manager. <a href="#" data-toggle="modal" data-target="#chill-out">:D</a></p>
```
Comment (dbgrandi) [inline views/socks.slim]: Does not open the link. Should be `<a href` instead of `<a src`

File: views/socks.slim [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+      article.row
+          p style="text-align:center; padding:0 20px;"
+            a href="https://charitywater.org/" target="_blank"
+              img src="/images/charity-drive.png" width="415"
+
+  section.podfile.container.tldr
+    article.row
+      .col-xs-12
+        div
+          h2 TLDR:
```
Comment (segiddins) [inline views/socks.slim]: `TL;DR:`

File: views/socks.slim [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+    		var leftPoint = wavePoints[n-1];
+        processing.vertex(leftPoint.x, leftPoint.y + overlapSines(leftPoint.x));
+        processing.vertex(p.x, p.y + overlapSines(p.x));
+    	}    
+    }
+    processing.vertex(document.width, 360)
+    processing.endShape()
+  }
+  
+
```
Comment (segiddins) [inline views/socks.slim]: missing trailing newline

**Comment by orta** [comment]
Thanks! Feedback addressed

**Comment by vaderdan** [comment]
there is problem on chrome and firefox - I can't see the water animation:
the problem is caused by the fact that 
`document.width;`
is not working on all browsers - for ex on chrome this returns `undefined`

you can use 
`document.body.clientWidth`
or if you prefer jQuery
`$(window).width()`

also on retina display the water animation is very pixelated
it is possible to resize it 2x with javascript and then scale it back with css like that here
http://jsfiddle.net/4JH75/

**Comment by orta** [comment]
awesome!

**Pull request merged**